content/rust/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Data-Types'
+Description: 'A brief description.'
```

Comment:
Make this desc a small introduction or definition what data types are

Reply:
Sorry i had not seen that

Reply:
@moha230 you still havent fixed the description